src/form-service.ts

```diff
@@ -707,6 +707,38 @@ async function loadFormElementDynamicOptions(
         }),
       )
     })(),
+    (async () => {
+      //Obtain all the Lookup ids that are featured on the elements on all forms
+      const formElementLookupsIds = forms.reduce((ids: number[], form) => {
+        formElementsService.forEachFormElement(form.elements, (el) => {
+          const lookupElement = typeCastService.formElements.toLookupElement(el)
+          if (lookupElement) {
+            if (lookupElement.isDataLookup && lookupElement.dataLookupId) {
+              ids.push(lookupElement.dataLookupId)
+            }
+            if (
+              lookupElement.isElementLookup &&
+              lookupElement.elementLookupId
+            ) {
+              ids.push(lookupElement.elementLookupId)
+            }
+          }
+        })
+        return ids
```

Comment:
what is the purpose of getting these if you don't use them?  You just need to know if one exists right?